resources/src/main/java/org/robolectric/res/ResourceIds.java

```diff
@@ -20,7 +20,7 @@ public static int getEntryIdentifier(int resId) {
     return resId & 0x0000FFFF;
   }
 
-  public static int makeIdentifer(int packageIdentifier, int typeIdentifier, int entryIdenifier) {
-    return packageIdentifier << 24 | typeIdentifier << 16 | entryIdenifier;
+  public static int makeIdentifier(int packageIdentifier, int typeIdentifier, int entryIdentifier) {
```

Comment:
Same remark here. I can deprecate `makeIdentifer()` and introduce a new `makeIdentifier()` method instead of renaming the existing one.